app/models/report.rb

```diff
@@ -29,6 +29,11 @@ class << self
       solr_fields: %w[id],
       sort: false, default: false, width: 100
     },
+    {
+      field: SolrDocument::FIELD_RELEASED_TO, label: "Released To",
```

Comment:
missing `.to_sym`?

Reply:
No, because the fields declared in `SolrDocument` are an inscrutable mix of strings and symbols. 😩  I was planning to make this more consistent in some follow-on work.